teams/deployment/membership.json

```diff
@@ -0,0 +1,44 @@
+{
+    "_comment": [
+        "Items of lists in this file are in alphabetical order. Please keep it on modification."
+    ],
+    
+    "maintainers": [
+        "DanielZhangQD",
```

Comment:
Also @DanielZhangQD @csuzhangxc you'd better be aware of this ongoing effort, if you think it is unreasonable to merge sig-tiup with sig-k8s (sincerely, we shouldn't name as sig-k8s), please comment here. Otherwise I will regard it as an acquiescence.

Reply:
@tisonkun I think it's not reasonable to merge the two sigs, the users, the background of the contributors are very different. WDYT @AstroProfundis

Reply:
@DanielZhangQD yep. From #527 I think the tiup community is in an early stage that we'd better leave it as is to grow faster, while sig-k8s (do you think "deployment" a proper name? at least we shouldn't name it "k8s") is more mature and has its own release plan, focusing domain, and audience.

Reply:
`sig-k8s` may not be that suitable, however, the users and contributors have been familiar with that name for several years, what about we keep that name for now and see if any opposed feedback from the community?

Reply:
@DanielZhangQD you convince me. I'll make a final suggestion to choose one of

* orchestration
* operator
* kubernetes (even expand the abbr.)

if you insist keep as is, since you're the maintainer, I'll respect your opinion.

Reply:
Separated two teams.

Now the original sig-k8s has one maintainer, I'd prefer to more maintainers to take the responsibility of growing the community. See also https://github.com/pingcap/community/pull/522#discussion_r690889094

May @csuzhangxc @mikechengwei qualified.

Reply:
@tisonkun Thanks for your advice!
We choose `kubernetes`, and please also add @csuzhangxc as the maintainer, thanks!

Reply:
Updated as e1dd2ee0060f28117809ef18769755c373ceecda